grunt/tasks/jasmine.js

```diff
@@ -21,7 +21,9 @@ module.exports = {
     return {
 
       'src': {
-        src: [], // actual src files are require'd in the *.spec.js files
+        src: [
+          'dist/cartodb.uncompressed.js',
+        ],
```

Comment:
If I remove it now, then `cdb._` doesn't get defined and there's a dependency on that [here](https://github.com/CartoDB/cartodb.js/blob/fix-test-suite/src/geo/gmaps/gmaps-torque-layer-view.js#L4).